api/v1alpha1/argocd_types.go

```diff
@@ -651,6 +651,8 @@ type KustomizeVersionSpec struct {
 type ArgoCDMonitoringSpec struct {
 	// Enabled defines whether workload status monitoring is enabled for this instance or not
 	Enabled bool `json:"enabled"`
+	// DisabledMetrics field can be used to enable or disable the collection of Metrics on Openshift
+	DisabledMetrics *bool `json:"disabledMetrics,omitempty"`
```

Comment:
```suggestion
	MetricsEnabled *bool `json:"metricsEnabled,omitempty"`
```
WDYT?

Reply:
As discussed internally, the metrics will be enabled by default so it makes sense to have it as `DisabledMetrics`.